src/handlers/slack/index.ts

```diff
@@ -1,27 +1,30 @@
 import serverlessExpress from "@vendia/serverless-express";
 import { app, expressReceiver, lockBot, prefix, url } from "./infra";
 import handleCommand from "./handle-command";
-import { getFirstParam, parseUnlock } from "./command-parsers";
+import { getFirstParam, getSecondParam, parseUnlock } from "./command-parsers";
 
 app.command(
   `/${prefix}locks`,
   handleCommand((command) => lockBot.locks(command.channel_id, command.team_id))
 );
 app.command(
   `/${prefix}lock`,
-  handleCommand((command) =>
-    lockBot.lock(
-      getFirstParam(command.text),
+  handleCommand((command) => {
+    const resourceName = getFirstParam(command.text);
+    const descriptionMessage = getSecondParam(command.text);
```

Comment:
I was thinking instead of `message` or `descriptionMessage` we could consistently call this `description` everywhere in the types/code/spec/docs etc.